cmd/algocfg/profileCommand.go

```diff
@@ -133,6 +134,24 @@ var listProfileCmd = &cobra.Command{
 	},
 }
 
+var printProfileCmd = &cobra.Command{
+	Use:   "print",
+	Short: "Print config.json to stdout.",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		cfg, err := getConfigForArg(args[0])
+		if err != nil {
+			reportErrorf("%v", err)
+		}
+		stdoutName := os.Stdout.Name()
+		err = codecs.SaveNonDefaultValuesToFile(stdoutName, cfg, config.GetDefaultLocal(), nil, true)
```

Comment:
nit: it seems like calling `os.Create` on `os.Stdout.Name()` can produce a permission error sometimes. I get the error `open /dev/stdout: permission denied` sometimes when running `go test` through my IDE on a small example which does this. This question might have relevant context: https://unix.stackexchange.com/questions/38538/bash-dev-stderr-permission-denied

Reply:
Disappointing to hear that this doesn't work on Mac, was hoping to avoid refactoring `SaveNonDefaultValuesToFile`.

Reply:
Refactored with an implementation that uses the normal writer pattern.